### PR TITLE
Fixing some typos and formatting on the GKE instructions page

### PR DESCRIPTION
**Pull request by richardsliu**
Fixed a few typos and formatting issues in the Kubeflow GKE instructions. Also added some missing steps during credentials creation.

### CODE REVIEW

**Comment by richardsliu** [comment]
@jlewi Please take a look.

**Comment by jlewi** [comment]
Is this ready for review? "/assign" it to someone when it is.
Looks like you are removing some of the code boxes; is that intentional?

**Comment by richardsliu** [comment]
I removed some of the newlines - I believe these cause Markdown to treat the lines in between as <p>, which is causing the bullet markers and list numbers to misalign.

**Comment by jlewi** [comment]
Should we
  * Put the URL http://<hostname>/_gcp_gatekeeper/authenticate in a code box?
  * Put <hostname> and <name> in bold?

screenshot

**Comment by jlewi** [comment]
Step 4. in Create OAuth Client id; can we put the environment variables in a code box so its easy to copy and paste?

**Comment by jlewi** [comment]
In the Create Kubeflow Deployment can you add back the code boxes. The code boxes make it easy to see the commands you need to run and copy and paste them.

Compare the following in your PR
create_deployment

To the current 
current

**Comment by richardsliu** [comment]
@jlewi Done. Netlify and github renders Markdown differently, so the preview versions didn't match before.

File: content/docs/started/getting-started-gke.md [hunk 1]
```diff
@@ -14,57 +14,55 @@ bref = "The Kubeflow project is dedicated to making deployments of machine learn
 
 Instructions for optimizing and deploying Kubeflow on GKE.
 
-Running on Kubeflow on GKE comes with the following advantages
+Running on Kubeflow on GKE comes with the following advantages:
```
Comment (jlewi) [inline content/docs/started/getting-started-gke.md]: delete "on"

Comment (jlewi) [reply content/docs/started/getting-started-gke.md]: -> Running Kubeflow on GKE

**Comment by richardsliu** [comment]
Done.

**Pull request merged**